packages/contracts-bedrock/test/setup/Setup.sol

```diff
@@ -156,7 +158,24 @@ contract Setup {
             args[0] = Executables.bash;
             args[1] = "-c";
             args[2] = string.concat(vm.projectRoot(), "/scripts/generate-l2-genesis.sh");
-            vm.ffi(args);
+            VmSafe.FfiResult memory result = vm.tryFfi(args);
```

Comment:
We can just use `Vm` here and remove the new `VmSafe` import. `Vm` inherits from `VmSafe`